src/bin-support.js

```diff
@@ -1,18 +1,32 @@
 'use strict';
 
-function runTransform(binRoot, transformName, paths) {
+function runTransform(binRoot, transformName, args) {
   const globby = require('globby');
   const execa = require('execa');
   const chalk = require('chalk');
   const path = require('path');
+  const yargs = require('yargs');
+
+  let parsedArgs = yargs.parse(args);
+  let paths = parsedArgs._;
+  let options = Object.keys(parsedArgs).reduce((acc, key) => {
+    if (!['_', '$0', 'help', 'version'].includes(key)) {
+      return acc.concat([`--${key}`, `${parsedArgs[key]}`]);
+    }
+    return acc;
+  }, []);
 
   return globby(paths)
     .then(paths => {
       let transformPath = path.join(binRoot, '..', 'transforms', transformName, 'index.js');
 
-      return execa('jscodeshift', ['-t', transformPath, '--extensions', 'js,ts', ...paths], {
-        stdio: 'inherit',
-      });
+      return execa(
```

Comment:
Possibly use:

```js
return execa(
  'jscodeshift',
  ['-t', transformPath, '--extensions', 'js,ts', ...paths],
  {
    stdio: 'inherit',
    env: {
      CODEMOD_CLI_ARGS: JSON.stringify(options),
    }
  }
);
```

Then add a utility function that can be used within the codemod itself to grab those options:

```js
const { getOptions } = require('codemod-cli');
const { getParser } = require('codemod-cli').jscodeshift;
module.exports = function transformer(file, api, options) {
  const j = getParser(api);
  const options = getOptions();

  return j(file.source);
}

```